rmf_site_format/src/workcell.rs

```diff
@@ -89,7 +90,14 @@ pub struct Mass(f32);
 
 #[derive(Serialize, Deserialize, Debug, Default, Clone)]
 #[cfg_attr(feature = "bevy", derive(Component))]
-pub struct Inertia {}
+pub struct Inertia {
+    pub ixx: f32,
+    pub ixy: f32,
+    pub ixz: f32,
+    pub iyy: f32,
+    pub iyz: f32,
+    pub izz: f32,
+}
 
 #[derive(Serialize, Deserialize, Debug, Default, Clone)]
 #[cfg_attr(feature = "bevy", derive(Bundle))]
```

Comment:
GitHub isn't letting me comment on the `origin: Pose` line, so this comment is a little bit off the mark.

I'd gently suggest changing the field name `origin` to `center` so that its relationship to "center of mass" is more clear. I don't consider this to be a big deal either way, but I think people will generally be more familiar with the concept of a "**center** of mass" than an "**origin** of the Inertial frame".

Reply:
https://github.com/open-rmf/rmf_site/pull/177/commits/aa70ca6c08b68314759143ce06ff60facd9ad465